.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+<!-- Thanks for opening a pull request! Please do not just delete this text. -->
+
+#### Summary
+<!--
+ Explain the **motivation** for making this change. What existing problem does the pull request solve? How can reviewers test this PR?
+
+Please remember to:
+- Pair the PR with an issue. If it is a new feature, the issue should proceed the PR and will have allowed sufficent time for discussions to take place. Pleases use
+issue tags such as "Closes #XYZ" or "Resolves sigstore/repo-name#XYZ".
+  [documentation](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
+- ensure your commits are signed-off, as sigstore uses the [DCO](https://en.wikipedia.org/wiki/Developer_Certificate_of_Origin) using `git commit -s`, or `git commit -s --amend` if you want to amend already existing commits
+
+Thank you :)
+-->
+
+##### Checklist
+
+- [ ] All commits are signed-off, using [DCO](https://en.wikipedia.org/wiki/Developer_Certificate_of_Origin)
+- [ ] All new code has docstrings and type annotations
+- [ ] Public facing changes are paired with documentation changes
+- [ ] Release note has been added to CHANGELOG.md if needed
+
+<!--
+Add a release note for each of the following conditions in CHANGELOG.md:
+
+* Model signature changes (additions, deletions, updates)
```

Comment:
the sigstore org template also had something about config changes. would that be applicable here? e.g. if we make a change to the default signing config?

https://github.com/sigstore/.github/blob/e313abb8c39be8bcbac48062e64cb52c861df254/pull_request_template.md

Reply:
I'd say so, the "model signature change" is more readable